src/webgl.cc

```diff
@@ -1384,7 +1384,7 @@ NAN_METHOD(GetShaderSource) {
   glGetShaderSource(shader, len, NULL, source);
 
   Local<String> str = JS_STR(source);
-  delete source;
+  delete [] source;
```

Comment:
what made you notice this, a memory leak?

Reply:
I read the clang warnings. 🙃